images/linux/scripts/installers/google-chrome.sh

```diff
@@ -39,11 +39,15 @@ wget "https://chromedriver.storage.googleapis.com/$LATEST_CHROMEDRIVER_VERSION/c
 unzip chromedriver_linux64.zip
 rm chromedriver_linux64.zip
 
-CHROMEDRIVER_BIN="/usr/bin/chromedriver"
+CHROMEDRIVER_DIR="/usr/local/share/chrome_driver"
+CHROMEDRIVER_BIN="$CHROMEDRIVER_DIR/chromedriver"
+
+mkdir -p $CHROMEDRIVER_DIR
 mv "chromedriver" $CHROMEDRIVER_BIN
 chown root:root $CHROMEDRIVER_BIN
```

Comment:
Do we really need this? Isn't packer builder already run from root user?

Reply:
Done.